backend/program/symlink.py

```diff
@@ -27,44 +28,38 @@ class Symlinker():
         host_path (str): The absolute path of the host mount.
         symlink_path (str): The path where the symlinks will be created.
     """
-    def __init__(self, _):
+    def __init__(self, *_):

```

Comment:
Hm, may not understand the need for the *args? Can you explain?

Reply:
I've never seen a requirement to initialize a class with an unused argument before. I understand we have an API we want to stick to, but why require it if we're not using it?  Currently, we have to initialize the class like `Symlink(None)`, which implies that there is some utility to the value.

Reply:
If its not being used anywhere, just toss it lol